.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@ repos:
         args: [--py37-plus]
 
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: 23.3.0
     hooks:
       - id: black
         args: [--target-version=py37]
```

Comment:
We can also remove this because Black will autodetect it:

> Black now tries to infer its `--target-version` from the project metadata specified in pyproject.toml (https://github.com/psf/black/pull/3219)

https://github.com/psf/black/releases/tag/23.1.0

```suggestion
```